hcipy/aperture/realistic.py

```diff
@@ -1325,3 +1325,167 @@ def func(grid):
 		segments = [segment_with_strut(segment) for segment in segments]
 
 	return func, segments
+
+def make_keck_aperture(normalized=True, with_spiders=False, with_segment_gaps=False, gap_padding=0, segment_transmissions=1, return_header=False, return_segments=False):
+    """
+    This code creates a keck-like aperture matching values used in vanKooten [2021 and 2022] as well as
```

Comment:
I'd add in an actual citation for these, so that they get formatted correctly in the compiled documentation.

Reply:
Done